pgxn/neon/walproposer.c

```diff
@@ -1176,6 +1190,7 @@ StartStreaming(Safekeeper *sk)
 	sk->state = SS_ACTIVE;
 	sk->active_state = SS_ACTIVE_SEND;
 	sk->streamingAt = sk->startStreamingAt;
+        UpdateDonorShmem(sk->wp);
```

Comment:
Let's add a comment here that we're only considering safekeepers in `SS_ACTIVE` to choose the donor. And we're calling `UpdateDonorShmem` here to update the donor because of that.

Reply:
Added